app/views/defence_requests/show_draft.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Check the request</h1>
+
+<p>Check the details below to make sure they are correct. Update any incorrect details, then submit the request.</p>
+
+<section class="detainee-details">
+  <h2><%= not_given_formatter(@defence_request, :detainee_name, "name_not_given") %></h2>
+
+  <%= render("defence_requests/show_partials/detainee_details") %>
+
+  <%= link_to("Change this", edit_defence_request_path(@defence_request, part: "detainee_details"), class: "edit-link") %>
+</section>
+
+
+<section class="detention-details">
+  <h2 class="line-above">Detention Details</h2>
+
+  <%= render("defence_requests/show_partials/detention_details") %>
+
+  <%= link_to("Change this", edit_defence_request_path(@defence_request, part: "detention_details"), class: "edit-link") %>
+</section>
+
+<div class="draft-actions">
```

Comment:
Missing the Custody details section:
Should be:
![screen shot 2015-06-02 at 16 55 57](https://cloud.githubusercontent.com/assets/722405/7940395/791f2230-0948-11e5-9627-bd5e484e5b17.png)

This is how it currently looks:
![screen shot 2015-06-02 at 16 56 13](https://cloud.githubusercontent.com/assets/722405/7940402/8759894e-0948-11e5-8f63-ffb9d1aef971.png)